File: DotNetBar WinForms Samples/VisualStudio2012StyleSample/VB/StartForm.vb
Imports System.ComponentModel
Imports System.Text

Namespace VisualStudio2012Style
	Partial Public Class StartForm
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private _LightThemeForm As LightThemeForm = Nothing
		Private Sub metroTileItem1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles metroTileItem1.Click
			If _DarkThemeForm IsNot Nothing Then
				_DarkThemeForm.Close()
			End If

			If _LightThemeForm Is Nothing Then
				Dim form As New LightThemeForm()
				form.StartPosition = FormStartPosition.CenterScreen
				form.Show()
				AddHandler form.FormClosed, AddressOf LightThemeFormClosed
				_LightThemeForm = form
			Else
				_LightThemeForm.Activate()
			End If
		End Sub
		Private Sub LightThemeFormClosed(ByVal sender As Object, ByVal e As FormClosedEventArgs)
			_LightThemeForm.Dispose()
			_LightThemeForm = Nothing
		End Sub

		Private _DarkThemeForm As DarkThemeForm = Nothing
		Private Sub metroTileItem2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles metroTileItem2.Click
			If _LightThemeForm IsNot Nothing Then
				_LightThemeForm.Close()
			End If

			If _DarkThemeForm Is Nothing Then
				Dim form As New DarkThemeForm()
				form.StartPosition = FormStartPosition.CenterScreen
				form.Show()
				AddHandler form.FormClosed, AddressOf DarkThemeFormClosed
				_DarkThemeForm = form
			Else
				_DarkThemeForm.Activate()
			End If
		End Sub
		Private Sub DarkThemeFormClosed(ByVal sender As Object, ByVal e As FormClosedEventArgs)
			_DarkThemeForm.Dispose()
			_DarkThemeForm = Nothing
		End Sub
	End Class
End Namespace

File: DotNetBar WinForms Samples/VisualStudio2012StyleSample/C#/StartForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace VisualStudio2012Style
{
    public partial class StartForm : Form
    {
        public StartForm()
        {
            InitializeComponent();
        }

        private LightThemeForm _LightThemeForm = null;
        private void metroTileItem1_Click(object sender, EventArgs e)
        {
            if (_DarkThemeForm != null)
                _DarkThemeForm.Close();

            if (_LightThemeForm == null)
            {
                LightThemeForm form = new LightThemeForm();
                form.StartPosition = FormStartPosition.CenterScreen;
                form.Show();
                form.FormClosed += new FormClosedEventHandler(LightThemeFormClosed);
                _LightThemeForm = form;
            }
            else
                _LightThemeForm.Activate();
        }
        void LightThemeFormClosed(object sender, FormClosedEventArgs e)
        {
            _LightThemeForm.Dispose();
            _LightThemeForm = null;
        }

        private DarkThemeForm _DarkThemeForm = null;
        private void metroTileItem2_Click(object sender, EventArgs e)
        {
            if (_LightThemeForm != null)
                _LightThemeForm.Close();

            if (_DarkThemeForm == null)
            {
                DarkThemeForm form = new DarkThemeForm();
                form.StartPosition = FormStartPosition.CenterScreen;
                form.Show();
                form.FormClosed += new FormClosedEventHandler(DarkThemeFormClosed);
                _DarkThemeForm = form;
            }
            else
                _DarkThemeForm.Activate();
        }
        void DarkThemeFormClosed(object sender, FormClosedEventArgs e)
        {
            _DarkThemeForm.Dispose();
            _DarkThemeForm = null;
        }
    }
}